.github/tailor.yaml

```diff
@@ -0,0 +1,10 @@
+rules:
+  - name: commit title
+    description: all commit titles must have a scope, be lowercase, and be no more than 50 characters
+    expression:  |-
+      .commits all((.message test "^([a-z/]+)|(\\*): [a-z \\-'./]+") and (.message test "^.{0,50}"))
```

Comment:
Would have rejected 9f7a25b2a7f40fe68a78e586cf71213425fb5703, 6b8539f219c95da1f8dfec58b7c5d861ebe4d11d, and 922e2973a40068169679d5ef8edd8cda5de2e3f6.

Reply:
I don't see why this would have failed the first. For the second two, I can update the rule to allow numbers and "Revert" at the beginning.

Reply:
It allows `*:` but not `config/*:`.

The uppercase `HTTP` in the second would also have been rejected.